src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -925,13 +926,16 @@ def delete_deployment_at_subscription_scope(cmd, deployment_name):
 
 def deploy_arm_template(cmd, resource_group_name,
                         template_file=None, template_uri=None, deployment_name=None,
-                        parameters=None, mode=None, rollback_on_error=None, no_wait=False, handle_extended_json_format=False):
+                        parameters=None, mode=None, rollback_on_error=None, no_wait=False,
+                        handle_extended_json_format=False, aux_subscriptions=None):
     if handle_extended_json_format:
         return _deploy_arm_template_unmodified(cmd.cli_ctx, resource_group_name, template_file, template_uri,
-                                               deployment_name, parameters, mode, rollback_on_error, no_wait=no_wait)
+                                               deployment_name, parameters, mode, rollback_on_error, no_wait=no_wait,
+                                               aux_subscriptions=aux_subscriptions)
 
     return _deploy_arm_template_core(cmd.cli_ctx, resource_group_name, template_file, template_uri,
-                                     deployment_name, parameters, mode, rollback_on_error, no_wait=no_wait)
+                                     deployment_name, parameters, mode, rollback_on_error, no_wait=no_wait,
+                                     aux_subscriptions=aux_subscriptions)
```

Comment:
So arm service will find tenant of those subs themselves?

Reply:
CLI will find all information including tenant id information according the subscription through [get_subscription()](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/_profile.py#L484) in [here](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/_profile.py#L535) and get the access token for the resource.